camshare/camshare/SignUpViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  SignUpViewController.swift
+//  camshare
+//
+//  Created by Janco Erasmus on 2020/02/18.
+//  Copyright © 2020 DVT. All rights reserved.
+//
+
+import UIKit
+import Firebase
+import FirebaseAuth
+
+class SignUpViewController: UIViewController {
+
+    @IBOutlet weak var firstNameTextField: UITextField!
+    @IBOutlet weak var lastNameTextField: UITextField!
+    @IBOutlet weak var emailTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var signUpButton: UIButton!
+    @IBOutlet weak var errorLabel: UILabel!
+    @IBOutlet weak var facebookSignUp: UIButton!
+    @IBOutlet weak var gmailSignUp: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        errorLabel.alpha = 0
+        customizeButtons()
+        // Do any additional setup after loading the view.
+    }
+    
+    @IBAction func signUpButtonTapped(_ sender: Any) {
+            //Validate Fields
+            let error = validateFields()
+            
+            if error != nil {
+                showError(error!)
+            }else {
+                //Create cleaned versions of the data
+                let firstName = firstNameTextField.text!.trimmingCharacters(in: .whitespacesAndNewlines)
```

Comment:
Does SwiftLint not tell you to not explicitly unwrap these?

Reply:
I did check that there will definitely be something in the variable before this piece of code

Reply:
But no, Swiflint does not complain